File: OfflineFetcher/app/src/main/java/de/jbamberger/offlinefetcher/db/entity/RedditPost.java
package de.jbamberger.offlinefetcher.db.entity;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

/**
 * @author Jannik Bamberger (dev.jbamberger@gmail.com)
 */

@Entity
public class RedditPost {

    @PrimaryKey
    private int id;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public static class Comment {

    }
}

File: OfflineClient/cached-api/src/main/java/de/jbamberger/api/provider/reddit/RedditPost.kt
package de.jbamberger.api.provider.reddit

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

/**
 * @author Jannik Bamberger (dev.jbamberger@gmail.com)
 */

@Entity
class RedditPost {

    @PrimaryKey
    var id: Int = 0

    class Comment
}
